app/components/dsfr_component/header_component/tool_link_component.rb

```diff
@@ -0,0 +1,20 @@
+class DsfrComponent::HeaderComponent::ToolLinkComponent < DsfrComponent::Base
+  def initialize(title:, path:, classes: [], html_attributes: {})
+    @title = title
+    @path = path
+
+    super classes: classes, html_attributes: html_attributes
```

Comment:
parens

Reply:
you’d prefer we stick to parentheses everywhere ? I'm currently using them only when necessary but I don’t have a strong opinion and I’m not very strict with myself on this rule

Reply:
super is special, and we have used parentheses before on all `super`s